ovh/ovh.go

```diff
@@ -22,6 +22,7 @@ const (
 	OvhEU        = "https://eu.api.ovh.com/1.0"
 	OvhCA        = "https://ca.api.ovh.com/1.0"
 	OvhUS        = "https://api.ovh.us/1.0"
+	OvhCloud     = "https://api.ovhcloud.com/1.0"
```

Comment:
OvhUS and OvhCloud is the same, we should merge this.

Reply:
We should keep the `OvhUS` constant (with the updated URL) as it might be used by some people.